### PR TITLE
Add translation files for Chinese.

### PR DESCRIPTION
**Pull request by nonwill**
Add initial translation support for Chinese.

### CODE REVIEW

File: src/QtUtils/src/Utils/LocalizationManager.cpp [hunk 1]
```diff
@@ -39,6 +39,7 @@ namespace Locale {
 
 const QString EN = QString::fromLatin1("en");
 const QString RU = QString::fromLatin1("ru");
+const QString ZH = QString::fromLatin1("zh");
```
Comment (AlienCowEatCake) [inline src/QtUtils/src/Utils/LocalizationManager.cpp:42]: This string is used for Qt translations too. But Qt has not qm file for "zh", it has separated "zh_CN" and "zh_TW" files. But it seems to LocalizationManager is not ready for such locale formats. I'll try to fix it

**Review by AlienCowEatCake — APPROVED** [review]

File: src/ImageViewer/resources/translations/imageviewer_zh.ts [hunk 2]
```diff
@@ -0,0 +1,1332 @@
...
+    </message>
+    <message>
+        <source>Next</source>
+        <comment>Short</comment>
+        <translation type="vanished">下一张</translation>
+    </message>
+    <message>
+        <source>Navigate</source>
+        <comment>Short</comment>
+        <translation type="vanished">Navigate</translation>
```
[the remark this answers is not in the document]
Comment (nonwill) [reply src/ImageViewer/resources/translations/imageviewer_zh.ts:159]: For shorter description here, ‘浏览’ might be the appropriate one. I'll check it and make an update later.

File: src/ImageViewer/resources/translations/imageviewer_zh.ts [hunk 3]
```diff
@@ -0,0 +1,1332 @@
...
+        <source>Value</source>
+        <translation>值</translation>
+    </message>
+</context>
+<context>
+    <name>MacToolBar</name>
+    <message>
+        <source>Previous</source>
+        <comment>Long</comment>
+        <translation type="vanished">上一张</translation>
```
[the remark this answers is not in the document]
Comment (nonwill) [reply src/ImageViewer/resources/translations/imageviewer_zh.ts:139]: > It looks like you're using a regular `lupdate` call.

Yes,it's.

I'll try to use the src/ImageViewer/update_translations.sh script for further works.

[the remark this answers is not in the document]
Comment (AlienCowEatCake) [reply src/ImageViewer/resources/translations/imageviewer_zh.ts:139]: It was fixed in [7c9c034](https://github.com/AlienCowEatCake/ImageViewer/commit/7c9c034474705a7ba0c9039b290a195b7c7171f9). Regular `lupdate` call should work as expected for now. `update_translations.sh` is not needed anymore

**Pull request merged**